### ISSUE TITLE
[Transferred from Web repo] New designs of top navigation bar for samsung galaxy fold devices.

### ISSUE THREAD

**Issue by prafulbbandre** [oppia/design-team#86]
**Platform**
Oppia Web 

**Describe the request**
According to the discussion on this issue https://github.com/oppia/oppia/issues/18337 @ConorMack  and @rfinch11 had advice to put the language selection to the hamburger menu but how should it look like in the hamburger menu is still not confirm so please provide a mock for it.
I might have miss it while searching if so Please corret me.

![image](https://github.com/oppia/design-team/assets/109658545/b3f6be6a-ec5b-4864-975c-1ec79dbf0bc2)
Currently the language selection is covering the oppia text and it is not looking good.

**Note**: this is also observed on Galaxy Fold dimensions: https://github.com/oppia/design-team/issues/121. It happens on multiple pages throughout the site, and the language and account buttons currently just disappear at like 355 px width (when they are supposed to just become smaller). This is due to the media query applied on the buttons:
![image](https://github.com/oppia/oppia/assets/154577754/2b6de8be-e73f-4c1e-abd8-0251a547f2d8)

The aim of this issue is to figure out how the navbar should work in a fully-responsive way across different devices.

**DESIGN APPROVAL CHECKLIST -- DO NOT EDIT.**

- [x] UX Design approved
- [x] UX Writing approved
- [ ] Product approved

_For a design project to be completed, it needs approval from UXD, UXW and PM. Please tag the following reviewers when your project is ready (at least one reviewer for each of the 3 bullet points below):_
- UXD: @rflore / @S4v8n (for Web projects)
- UXW: @ccc253 / @tamiorendain 
- PM: @seanlip

**Comment by kmuir1991** [comment]
@seanlip does this need a UXW? Seems to be UXD only.

**Comment by seanlip** [comment]
Hi @kmuir1991, I agree it's mostly UXD. However, as a matter of policy, I suggest assigning a UXW to any issue that has a UXD, so that there is a point of contact for any UXW questions that come up. If the UXW is subsequently still unaffected, then UXW approval on the final designs should be trivial. (But one could imagine changes to the wording arising from a design solution, e.g. if we reformat the dropdown menus to accommodate the language selector, say.)

Does the above approach sound good to you? That way, you won't need to make the triaging decision each time and it'll be quicker to audit that each in-progress issue has a UXW assignee.

**Comment by seanlip** [comment]
Also assigning @ConorMack as UXD since I believe he's looking into this (@ConorMack please feel free to post any updates to this thread, thanks!)

**Comment by kmuir1991** [comment]
@seanlip yep, that sounds fine to me! It sounds like the rest of the UXW team is okay with me assigning them some of these remaining projects, so I'm going to give this one to @tydesignit

**Comment by S4v8n** [comment]
Hi @tamiorendain @davishedrick @aileenyesand,  Hope you are all well.  We need a lead UXW to review this project for UXW lead approval.  Once there are no changes required, please check mark the box "UXW Lead approval" in this project. If revisions are required please add comments on the design file for the designer respond.  Thank you in advance.

**Comment by tamiorendain** [comment]
Hi @S4v8n, got it! Added this to our task list

**Comment by davishedrick** [comment]
Hey @S4v8n we're prepared to sign off since there really isn't much language to review. Thanks!

**Comment by S4v8n** [comment]
Great! Thank you for the update. On Feb 12, 2025, at 4:41 AM, davishedrick ***@***.***> wrote:﻿
Hey @S4v8n we're prepared to sign off since there really isn't much language to review. Thanks!

**Comment by seanlip** [comment]
Note: https://github.com/oppia/oppia/issues/22050#issuecomment-2708808751 is a duplicate of this
